### PR TITLE
Implement knowledge nuggets feature

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-465, L2B-466, L2B-467

### CODE REVIEW

File: packages/config/src/layer2s/types/Layer2.ts [hunk 1]
```diff
@@ -31,6 +32,9 @@ export interface Layer2 {
   /** Links to recent developments */
   /** Milestones achieved by the project */
   milestones?: Milestone[]
+  /** List of knowledge nuggets */
+  /** Useful articles worth to read*/
+  knowledgeNuggets?: KnowledgeNugget[]
```
Comment (sz-piotr) [inline packages/config/src/layer2s/types/Layer2.ts:37]:
```suggestion
  /** Links to recent developments, milestones achieved by the project */
  milestones?: Milestone[]
  /** List of knowledge nuggets: useful articles worth reading */
  knowledgeNuggets?: KnowledgeNugget[]
```

File: packages/frontend/src/components/project/KnowledgeNuggets.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+import { ArrowRightIcon } from '../icons/Arrows'
+import { OutLink } from '../OutLink'
+
+export interface KnowledgeNuggetsProps {
+  knowledgeNuggets?: KnowledgeNugget[]
+}
+
+export function KnowledgeNuggets(props: KnowledgeNuggetsProps) {
+  const { knowledgeNuggets } = props
```
Comment (sz-piotr) [inline packages/frontend/src/components/project/KnowledgeNuggets.tsx:12]:
```suggestion
export function KnowledgeNuggets({ knowledgeNuggets }: KnowledgeNuggetsProps) {
```

File: packages/frontend/src/components/project/KnowledgeNuggets.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      </div>
+    </div>
+  )
+}
+
+interface KnowledgeNuggetProps {
+  knowledgeNugget: KnowledgeNugget
+}
+
+function KnowledgeNuggetItem(props: KnowledgeNuggetProps) {
```
Comment (sz-piotr) [inline packages/frontend/src/components/project/KnowledgeNuggets.tsx:34]: As we have a single file per component everywhere (or almost everywhere) else please extract this too. There can still be just a single story for the list of knowledge nuggets.

File: packages/frontend/src/components/project/Milestones.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ export function Milestones({ milestones, className }: MilestonesProps) {
   return (
     <div
       className={cx(
-        'Milestones py-12 px-4 md:px-0 bg-gray-900 md:bg-transparent',
+        'Milestones px-4 md:px-0 bg-gray-900 md:bg-transparent',
```
Comment (sz-piotr) [inline packages/frontend/src/components/project/Milestones.tsx:22]: Is the `Milestones` class still relevant?

**Review by sz-piotr — APPROVED** [review]

**Review by sz-piotr — CHANGES_REQUESTED** [review]
Sorry, I wanted to request changes instead

File: packages/frontend/src/components/project/KnowledgeNuggets.tsx (hunk 3 above)
Comment (torztomasz) [reply packages/frontend/src/components/project/KnowledgeNuggets.tsx:32]: Okay, sure :)

File: packages/frontend/src/components/project/KnowledgeNuggetItem.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div className="p-4">
+      <KnowledgeNuggetItem {...props} />
+    </div>
+  )
+}
+
+KnowledgeNugget.args = {
+  knowledgeNugget: knowledgeNuggetExample,
+}
```
Comment (sz-piotr) [inline packages/frontend/src/components/project/KnowledgeNuggetItem.stories.tsx:29]: Why is this here? I don't think we use anything like this anywhere.

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Review by sz-piotr — APPROVED** [review]
